sycl/CMakeLists.txt

```diff
@@ -129,9 +129,20 @@ add_custom_target( sycl-toolchain
   COMMENT "Building SYCL compiler toolchain..."
 )
 
-add_subdirectory( test )
-add_subdirectory( unittests )
-add_subdirectory( tools )
+if (NOT DEFINED LLVM_INCLUDE_TESTS)
+  set(LLVM_INCLUDE_TESTS ON)
+endif()
```

Comment:
Aligns testing and unittests with other LLVM projects CMake usage.

Reply:
Add this as a comment too.

Reply:
The commit contains a comment that SYCL CMake is now stronger aligned with how the other LLVM projects are set up.

Other LLVM projects I checked do not comment on where they copied and adapted from other LLVM projects though it is clear that they are aligned.